xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -237,6 +237,14 @@ namespace PVR
      */
     std::string EpisodeName(void) const { return m_strShowTitle; };
 
+    /*!
+    * @brief Change the genre of this recording.
```

Comment:
please describe in more detail what the member function actually does, esp. the fallback strategy.